deploy/docker-compose-example.yml

```
@@ -26,7 +26,7 @@ services:
       - ./docker-example/edumfa-setup.sh:/opt/edumfa/user-scripts/01-setup.sh:ro
       - ./docker-example/edumfa-config.conf:/opt/edumfa/policy.conf:ro
       - ./docker-example/edumfa.cfg:/opt/edumfa/edumfa.cfg:ro
-      - ./docker-example/config:/etc/edumfa/:rw
+      - edumfa-data:/etc/edumfa/:rw
```

```suggestion
      - edumfa-config:/etc/edumfa/:rw
```
`-config` might better here